src/hub.html

```diff
@@ -597,9 +600,20 @@
                         icon-button="image: snap.png; hoverImage: snap-hover.png; activeImage: snap.png; activeHoverImage: snap-hover.png"
                         visibility-on-content-type="contentType: video/mp4; contentSubtype: video-camera; visible: false;"
                         scale="0.075 0.075 0.075"
-                        position="0 0.225 0"
+                        position="0 0.125 0"
                         class="video-snap-button"
                     ></a-entity>
+
+                    <a-entity
+                    is-remote-hover-target
+                    tags="singleActionButton: true; isHoverMenuChild: true;"
+                    sprite
+                    icon-button="image: snap.png; hoverImage: snap-hover.png; activeImage: snap.png; activeHoverImage: snap-hover.png"
```

Comment:
If we're going to keep this media hover button functionality, we should change this icon to something other than snap.png.